docs/input.md

```diff
@@ -206,30 +206,15 @@ for the given input, **within** the container. Data examples are a prime source
 for your users to understand how inputs should look like and be formatted.
 
 ```yaml
-example: /samples/input_name.csv
+example: /in/input_name.csv
 ```
 
-#### `quality`
-
-The `quality` field is an optional field, that contains various sub-fields.
-These text-based fields can be used to specify data quality requirements for the
-input data.
-The quality field can contain one or more of the child fields, but cannot be empty.
-
-```yaml
-quality:
-  completeness: Describes the expectations of present variables and measurements.
-  accuracy: | 
-    Describes if the tool has expectations of minimum required accurary.
-    This can involve measurement accuracy, but also expected scaling.
-  validity: | 
-    Describes which format requirements the tool has, to recognize the passed in
-    files as valid data inputs. 
-```
-
-There are additional dimensions to data quality, consistency, timeliness and 
-uniqueness. These dimensions do not apply here as general catgories.
-
+It is considered good practice to add example data and example parameterizaitons
```

Comment:
*parametrization

Reply:
I always use this spelling. You can use 'parameterization'  and 'parametrization'